domain/domain.go

```diff
@@ -503,7 +503,7 @@ func (do *Domain) Init(ddlLease time.Duration, sysFactory func(*Domain) (pools.R
 	sysFac := func() (pools.Resource, error) {
 		return sysFactory(do)
 	}
-	sysCtxPool := pools.NewResourcePool(sysFac, 2, 2, resourceIdleTimeout)
+	sysCtxPool := pools.NewResourcePool(sysFac, 4, 10, resourceIdleTimeout)
```

Comment:
Why change it?

Reply:
I misunderstand here, actually the maxCap of sysCtxPool need to be equal to the count of the ddl worker. So 2 is ok. I'll change it back.